Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM helsinkitest/python:2.7-slim as appbase
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /app
+RUN mkdir /entrypoint
+
+COPY --chown=appuser:appuser requirements*.txt /app/
+
+RUN apt-install.sh \
+    build-essential \
+    libpq-dev \
+    && pip install --no-cache-dir \
+    -r /app/requirements.txt \
+    #&& pip install --no-cache-dir \
+    #-r /app/requirements-prod.txt \
```

Comment:
We probably should add `requirements-prod.in` which includes `uwsgi`

Reply:
I'll add it.

Reply:
Added the requirements and changed Dockerfile accordingly.